### PR TITLE
BFD-704: Try to fix intermittent bfd-server SocketException test failures

### PR DESCRIPTION
**Pull request by karlmdavis**
### Change Details

This one is a bit harder to debug, just because it's less frequent, but I think I may have tracked down the issue. I think it's similar to the other `SocketException` errors that were just fixed in `bfd-pipeline`: the server was binding to a different host/interface than we were trying to connect to in our tests.

### Acceptance Validation

1. Does this look correct to you?
2. Is the build passing?

### Feedback Requested

1. What's your spirit animal?

### External References

- [BFD-704](https://jira.cms.gov/browse/BFD-704)

### Security Implications

This change has no known security implications.

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by mackec1 — APPROVED** [review]

File: apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/AppConfiguration.java [hunk 1]
```diff
@@ -8,7 +8,13 @@
 
 /** Models the configuration options for the launcher. */
 public final class AppConfiguration implements Serializable {
-  private static final long serialVersionUID = -8134110339292440841L;
+  private static final long serialVersionUID = 1L;
```
Comment (rtravitz) [inline apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/AppConfiguration.java:11]: What's the reason for this change?

**Review by rtravitz — APPROVED** [review]
Ran locally okay 👍

**Review by mackec1 — APPROVED** [review]

**Review by mackec1 — APPROVED** [review]

**Review by cbrunefearless — APPROVED** [review]
lgtm

File: apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/AppConfiguration.java (hunk 1 above)
Comment (karlmdavis) [reply apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/AppConfiguration.java:11]: Good question! Having computed serialization IDs provides a false sense of security, since nothing verifies them. (We also only use serialization for the test cases, so it doesn't matter, either way.)
